devspace.yaml

```
@@ -345,7 +345,7 @@ dev:
     labelSelector:
       app.kubernetes.io/instance: vantage6-auth
     ports:
-      - port: 8080:8080
+      - port: 7680:7680
```

```suggestion
      - port: 7680:7680
         bindAddress: localhost
      - port: 7680:7680
         bindAddress: ${PORTS_BIND_ADDRESS}
```

To make it work for both docker k8s and microk8s